site/src/sphinx/server-sse.rst

```diff
@@ -49,3 +53,37 @@ use the annotation.
     public Publisher<ServerSentEvent> sse() {
         return Flux.just(ServerSentEvent.ofData("foo"), ServerSentEvent.ofData("bar"));
     }
+
+Adjusting the request timeout
+-----------------------------
+
+The events may being sent for a longer time depending on the application. Even it can continue infinitely,
+for example streaming stock quotes. But the server allows receiving a request and sending the corresponding
+response completely only within the amount of time, which is configured as the request timeout property.
+Even if you do not configure the property, it is configured to ``10`` seconds by default.
```

Comment:
'But the server .... seconds by default.' could be simplified like this:

```
Such a long running stream may be terminated prematurely because Armeria has
the default request timeout of ``10`` seconds, i.e. your stream will be broken
after 10 seconds at most.
```